nifi-nar-bundles/nifi-slack-bundle/nifi-slack-processors/src/main/resources/docs/org.apache.nifi.processors.slack.PostSlack/additionalDetails.html

```diff
@@ -0,0 +1,91 @@
+            a <a href="https://api.slack.com/bot-users">Bot User</a> to the app, and to install
+            the app in your Slack workspace. After the app installed, you can get the Bot User's
+            OAuth Access Token that will be needed to authenticate and authorize your PostSlack
+            processor to Slack.
+        </p>
+        <p>
+            PostSlack provides 2 main functions / modes:
+        </p>
+        <ul>
+            <li>sending text message without FlowFile upload</li>
+            <li>sending text message with FlowFile upload</li>
+        </ul>
+        <p>
+            These 2 functions / modes use different Slack Web API methods which have different parameters
+            (for further info you can see <a href="https://api.slack.com/methods/chat.postMessage">chat.postMessage</a>
+            and <a href="https://api.slack.com/methods/files.upload">files.upload</a>).
+            Due to this reason, each mode has its own processor properties beyond the common ones. Filling in
+            the not applicable properties for a function will not cause any problems, they will just not have any effect.
+            You can switch between the 2 modes by setting the 'Upload FlowFile' property to Yes/No.
+        </p>
+        <p>
+            PostSlack properties used when 'Upload FlowFile' has been set to 'No' (default):
+        </p>
+        <ul>
+            <li><i>Post Message URL</i></li>
+            <li>Access Token</li>
+            <li>Channel</li>
+            <li>Text</li>
+            <li><i>Username</i></li>
+            <li><i>Icon Emoji</i></li>
+            <li><i>Icon URL</i></li>
+            <li><i>Dynamic Properties (see below)</i></li>
+        </ul>
+        <p>
+            PostSlack properties used when 'Upload FlowFile' has been set to 'Yes':
+        </p>
+        <ul>
+            <li><i>File Upload URL</i></li>
+            <li>Access Token</li>
+            <li>Channel</li>
+            <li>Text</li>
+            <li><i>File Title</i></li>
+            <li><i>File Name</i></li>
+            <li><i>File Mime Type</i></li>
+        </ul>
+        <p>
+            <a href="https://api.slack.com/docs/message-attachments">Slack content and link attachments</a> can also be
+            added to the message. Please note that this kind of Slack message attachments does not involve the file
+            upload itself, but rather contain links to external resources or internal resources already uploaded
+            to Slack. Please also note that this functionality does not work together with file upload
+            (so it can only be used when 'Upload FlowFile' has been set to 'No').
+            The Dynamic Properties can be used to specify these Slack message "attachments" as JSON snippets.
+            Each property value will be converted to JSON and will be added to the array of attachments in the JSON
+            payload being sent to Slack.
```

Comment:
It would be very useful to include an example JSON snippet here, I used the link as reference and missed that I needed to extract the snippet from the `attachments` array in order to use it as the dynamic property value.

Reply:
I'll add the example.